src/tools/run-make-support/src/lib.rs

```diff
@@ -468,6 +480,12 @@ macro_rules! impl_common_helpers {
                 self.cmd.run_fail()
             }
 
+            /// Run the constructed command and return its output no matter what.
```

Comment:
Suggestion: I think we should warn to strongly prefer ``[`run`]`` or ``[`run_fail`]``, and that this should probably say something more like "Run the command returning its output, intentionally ignoring exit status".